src/lib/constants.ts

```diff
@@ -169,3 +169,64 @@ export const LSP6_DEFAULT_PERMISSIONS = {
 
 export const LSP6_ALL_PERMISSIONS =
   '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+
+export const COMMON_ABIS = {
+  supportsInterface: [
+    {
+      inputs: [
+        {
+          internalType: 'bytes4',
+          name: 'interfaceId',
+          type: 'bytes4',
+        },
+      ],
+      name: 'supportsInterface',
+      outputs: [
+        {
+          internalType: 'bool',
+          name: '',
+          type: 'bool',
+        },
+      ],
+      stateMutability: 'view',
+      type: 'function',
+    },
+  ],
+};
+
+export const IPFS_GATEWAY_BASE_URL = 'https://ipfs.io/ipfs/'; // 'https://2eff.lukso.dev/ipfs/' not working;
```

Comment:
Is there not already a default ipfs gateway value somewhere? Why not use that?

Reply:
for some reason, the cloudflare IPFS does not work sometimes for assets on l16. had this issue with Jean several times, so i just thought I go with how we handle it in the wallet.